_pages/en_US/troubleshooting.txt

```diff
@@ -514,6 +545,12 @@ Your Luma3DS version is outdated. Download the latest release of [Luma3DS](https
 {% endcapture %}
 <details>{{ compat | markdownify }}</details>
 
+{% capture compat %}
+<summary><u>"Unable to mount CTRNAND or load the CTRNAND FIRM. Please use an external one."</u></summary>
+There are a number of reasons as to why this could be happening. In any case, this error should be fixed by following the [CTRTransfer](ctrtransfer) guide.
```

Comment:
should be fixed -> can usually be fixed? slightly different nuance, personal stylistic choice